src/mongocxx/options/find_one_and_update.hpp

```diff
@@ -27,6 +26,17 @@ MONGOCXX_INLINE_NAMESPACE_BEGIN
 namespace options {
 
 ///
+/// Argument for find_one_and_update and find_one_and_replace to determine
+/// whether to return the document before or after modification.
+///
+/// @see https://docs.mongodb.org/manual/reference/method/db.collection.findAndModify/#return-data
+///
+enum class return_document {
```

Comment:
Hmm... so should we just delete the comment, and call it good? I'm fine with that! It does seem weird for either of these classes to rely on the other's header.